src/Features/Core/Portable/Shared/Extensions/ISymbolExtensions_2.cs

```diff
@@ -182,14 +182,27 @@ private static string GetDocumentation(ISymbol symbol, CancellationToken cancell
         {
             switch (symbol)
             {
-                case IParameterSymbol parameter: return parameter.ContainingSymbol.OriginalDefinition.GetDocumentationComment(cancellationToken: cancellationToken).GetParameterText(symbol.Name);
+                case IParameterSymbol parameter: return GetParameterDocumentation(parameter, cancellationToken);
                 case ITypeParameterSymbol typeParam: return typeParam.ContainingSymbol.GetDocumentationComment(cancellationToken: cancellationToken).GetTypeParameterText(symbol.Name);
                 case IMethodSymbol method: return GetMethodDocumentation(method);
                 case IAliasSymbol alias: return alias.Target.GetDocumentationComment(cancellationToken: cancellationToken).SummaryText;
                 default: return symbol.GetDocumentationComment(cancellationToken: cancellationToken).SummaryText;
             }
         }
 
+        private static string GetParameterDocumentation(IParameterSymbol parameter, CancellationToken cancellationToken)
+        {
+            var containingSymbol = parameter.ContainingSymbol;
+            if (containingSymbol.ContainingSymbol.IsDelegateType())
+            {
+                // containingSymbol is the Invoke method of the delegate, so we need to go one level up and take the method's containing symbol.
+                containingSymbol = containingSymbol.ContainingSymbol;
```

Comment:
Are we sure th is currect?  How do you know it's the Invoke mehtod?  What if it's hte BeginInvoke method (or any other number of methods that are on delegates)?  can you actually validate that the containing symbol is the 'Invoke' method?